### PR TITLE
Bug 560883 - [Passage] publish org.eclipse.passage.loc.yars.api

### PR DESCRIPTION
**Pull request by ruspl-afed**
Extracted "org.eclipse.passage.loc.report.feature" feature

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: features/org.eclipse.passage.loc.report.feature/build.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2018, 2020 ArSysOp and others
```
Comment (eparovyshnaya) [inline features/org.eclipse.passage.loc.report.feature/build.properties]: just 2020

File: features/org.eclipse.passage.loc.report.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	Copyright (c) 2018, 2020 ArSysOp and others
```
Comment (eparovyshnaya) [inline features/org.eclipse.passage.loc.report.feature/feature.xml]: Here too: just 2020

File: features/org.eclipse.passage.loc.report.feature/build.properties (hunk 1 above)
Comment (ruspl-afed) [reply features/org.eclipse.passage.loc.report.feature/build.properties]: done

File: features/org.eclipse.passage.loc.report.feature/feature.xml (hunk 2 above)
Comment (ruspl-afed) [reply features/org.eclipse.passage.loc.report.feature/feature.xml]: done

**Pull request merged**